orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/dcos/DcosContainerFinder.groovy

```diff
@@ -20,7 +20,10 @@ class DcosContainerFinder {
       if (!image) {
         throw new IllegalStateException("No image found in context for pattern $imageDescription.pattern.")
       } else {
-        imageDescription = [registry: image.registry, tag: image.tag, repository: image.repository, imageId: buildImageId(image.registry, image.repository, image.tag)]
```

Comment:
The original line is assigning imageDescription to a new map with those key/value pairs, but it has imageId in it, so I don't know why that wouldn't be set correctly. Perhaps it was another field that was coming from the original "operation.docker.image" that imageDescription is originally set to?